<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img_dir_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>561</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>图片文件夹</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dir_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>561</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>标签文件夹</string>
    </property>
   </widget>
   <widget class="QToolButton" name="img_dir_input_botton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>37</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="label_dir_input_botton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>37</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="do_botton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QComboBox" name="img_format_box">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.jpg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.jpeg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.PNG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.JPG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.JPEG</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="img_format_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>图片格式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_format_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>标签格式</string>
    </property>
   </widget>
   <widget class="QComboBox" name="label_format_box">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.xml</string>
     </property>
    </item>
   </widget>
   <widget class="QToolButton" name="save_dir_input_botton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>37</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="save_dir_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>561</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>保存文件夹</string>
    </property>
   </widget>
   <widget class="QLabel" name="save_dir_input_edit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_dir_input_edit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_dir_input_edit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>save_dir_label</zorder>
   <zorder>img_format_label</zorder>
   <zorder>img_dir_label</zorder>
   <zorder>label_dir_label</zorder>
   <zorder>img_dir_input_botton</zorder>
   <zorder>label_dir_input_botton</zorder>
   <zorder>do_botton</zorder>
   <zorder>img_format_box</zorder>
   <zorder>label_format_label</zorder>
   <zorder>label_format_box</zorder>
   <zorder>save_dir_input_botton</zorder>
   <zorder>save_dir_input_edit</zorder>
   <zorder>img_dir_input_edit</zorder>
   <zorder>label_dir_input_edit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
